osquery/core/CMakeLists.txt

```diff
@@ -68,20 +68,22 @@ ADD_OSQUERY_TEST_CORE(
   "${CMAKE_CURRENT_LIST_DIR}/database/tests/in_memory_database_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/base64_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/conversions_tests.cpp"
+  "${CMAKE_CURRENT_LIST_DIR}/tests/diff_results_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/error_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/exptected_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/flags_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/process_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/query_tests.cpp"
+  "${CMAKE_CURRENT_LIST_DIR}/tests/scheduled_query_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/status_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/system_test.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/tables_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/time_tests.cpp"
   "${CMAKE_CURRENT_LIST_DIR}/tests/watcher_tests.cpp"
 )
 
-if(POSIX) 
-  ADD_OSQUERY_TEST_CORE(
+if(POSIX)
+  ADD_OSQUERY_TEST(TRUE
```

Comment:
What was the problem with ADD_OSQUERY_TEST_CORE?
I suppose it should be the same